.drone.yml

```diff
@@ -2391,9 +2395,11 @@ steps:
   - rm -r -force grafana-enterprise
   - cp grabpl.exe C:\App\grabpl.exe
   - rm -force grabpl.exe
-  - C:\App\grabpl.exe init-enterprise C:\App\grafana-enterprise
+  - C:\App\grabpl.exe init-enterprise C:\App\grafana-enterprise master
```

Comment:
Why did this become `master`? 🤔

Reply:
There was one more reference I had missed, thanks.